docs/plugins/best_practices.md

```diff
@@ -0,0 +1,175 @@
+# Best practices when developing napari plugins
+
+There are a number of good and bad practices that may not be immediately obvious
+when developing a plugin.  This page
+
+## Do
+
+- **Update your README to make it immediately clear what your plugin does in a
+  few sentences** 
+
+    This is your elevator pitch. You'll loose a lot of potential users quickly
+    if they have to look in the source code, or look through a large
+    documentation page, just to know what the gist of the plugin is.
+
+- **Use images, gifs, or movies to *show* what your plugin does**. 
+
+  Images and movies are *extremely* useful for conveying the usefulness of your
+  plugin. Both github and napari hub make it easy to embed videos in your
+  readme. For example, if you have a dock widget plugin that plots something as
+  the user interacts with the current layer, capture a screengrab and put it in
+  your readme.
```

Comment:
Suggest some (cross platform) tools to do screen grabbing.

This could include something like e.g. licecap https://www.cockos.com/licecap/

And even suggesting tools that add the buttons you press when recording if this makes sense for the plugin